pytensor/tensor/basic.py

```diff
@@ -760,7 +760,16 @@ def switch(cond, ift, iff):
     """if cond then ift else iff"""
 
 
-where = switch
+def where(cond, ift=None, iff=None):
+    # Raise an error if only one of ift or iff is passed
+    if (ift is None and iff is not None) or (ift is not None and iff is None):
```

Comment:
May be a bit cleaner if you check the common case first (both not none) and else just check they are both None?

```python
if ift is not None and iff is not None:
  ... # switch
elif ift is None and iff is None:
  ... # nonzero
else:
  ... # raise
```